mappings/net/minecraft/entity/mob/ZombiePigmanEntity.mapping

```diff
@@ -1,4 +1,7 @@
 CLASS atm net/minecraft/entity/mob/ZombiePigmanEntity
+	CLASS net/minecraft/entity/mob/ZombiePigmanEntity$a FollowPlayerIfAngryGoal
```

Comment:
A few places here with Knit remapping the outer class name here

Reply:
Fixed